specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/blob.json

```diff
@@ -917,6 +1002,108 @@
           }
         },
         "description": "The list of blob containers."
+      },      
+      "BlobServiceProperties": {
+        "properties": {
+        "properties": {
+          "properties": {
+            "cors": { 
+              "$ref": "#/definitions/CorsRules",
+              "description": "Specifies CORS rules for the Blob service. You can include up to five CorsRule elements in the request. If no CorsRule elements are included in the request body, all CORS rules will be deleted, and CORS will be disabled for the Blob service."
+            },
+            "defaultServiceVersion": {
+              "type": "string",
+              "description": "DefaultServiceVersion indicates the default version to use for requests to the Blob service if an incoming request’s version is not specified. Possible values include version 2008-10-27 and all more recent versions."
+            },
+            "deleteRetentionPolicy": {
+              "$ref": "#/definitions/DeleteRetentionPolicy",
+              "description": "The blob service properties for soft delete."
+            }
+          },
+          "x-ms-client-flatten": true,
+          "x-ms-client-name": "BlobServiceProperties",
+          "description": "The properties of a storage account’s Blob service."
+        }
+        },
+        "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+        ],
+        "description": "The properties of a storage account’s Blob service."
+      },      
+      "DeleteRetentionPolicy": {
+        "properties": {
+          "enabled": {
+            "type": "boolean",
+            "description": "Indicates whether DeleteRetentionPolicy is enabled for the Blob service."
+          },
+          "days": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 365,
```

Comment:
>365 [](start = 23, length = 3)

just a note - in a leap year there are 366 days. Does it matter in your case?

Reply:
365 is intended max value as specified by underlying data-plane API. Thanks for noting!